client/src/components/ratings/sub-components/ReviewTile.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default function ReviewTile({review}) {
+
+  const starRater = function() {
+    var stars = '';
+    for (var i = 0; i < review.rating; i++) {
+      stars += '*';
+    }
+    return stars;
+  };
+
+  const dateConverter = function(dateString) {
+    let date = new Date(dateString);
+    let mm = date.getMonth() + 1;
+    let dd = date.getDate();
+    let yyyy = date.getFullYear();
+    return `${mm}/${dd}/${yyyy}`;
+  };
+
+  return (
+    <div>
+      {console.log(review)}
```

Comment:
do you still need this? should delete later if so. maybe add todo?

Reply:
Bad habit of mine to leave in old logs. Will clean up